src/LoginAuth0.js

```diff
@@ -68,7 +68,8 @@ class LoginAuth0 extends Component {
               }
             });
           });
-        } else {
+        }
+        else {
```

Comment:
can we do this in one line ? just a style thing

Reply:
You mean with a ternary operator or just like it was?

Reply:
no sorry,  i meant
```
} else {
```
instead of 
```
} 
else {
```

Reply:
no big deal though